.github/workflows/runAqa.yml

```diff
@@ -0,0 +1,193 @@
+      id: argparse
+    - name: Output log
+      if: failure()
+      # Store the contents of log.txt into an environment variable and escape characters to preserve newlines and other symbols.
+      run: |
+        log=$(cat log.txt)
+        log="${log//'%'/'%25'}"
+        log="${log//$'\n'/'%0A'}"
+        log="${log//$'\r'/'%0D'}"
+        log="${log/$'`'/'\`'}"
+        echo ::set-output name=log::$log
+      id: output_log
+    - name: Create error comment
+      if: failure()
+      uses: actions/github-script@v3
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          comment_body = `
+          @${{ github.actor }}
+          \`\`\`
+          ${{ steps.output_log.outputs.log }}
+          \`\`\`
+          No builds were started.
+          `;
+          github.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: comment_body
+          })
+    - name: Get workflow run info
+      run: |
+        echo ::set-output name=url::$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
+        echo ::set-output name=id::$GITHUB_RUN_ID
+      id: workflow_run_info
+    - name: Create success comment
+      uses: actions/github-script@v3
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          comment_body = `
+          @${{ github.actor }} Build(s) started with the following parameters:
+          - sdk_resource: ${{ steps.argparse.outputs.sdk_resource }}
+          - build_list: ${{ steps.argparse.outputs.build_list }}
+          - target: ${{ steps.argparse.outputs.target }}
+          - platform: ${{ steps.argparse.outputs.platform }}
+          - jdk_version: ${{ steps.argparse.outputs.jdk_version }}
+          - jdk_impl: ${{ steps.argparse.outputs.jdk_impl }}
+
+          Workflow Run ID: [${{ steps.workflow_run_info.outputs.id }}](${{ steps.workflow_run_info.outputs.url }})
+          `;
+          github.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: comment_body
+          })
+    - name: Echo parameters
+      run: |
+        echo build_parameters: ${{ steps.argparse.outputs.build_parameters }}
+        echo sdk_resource: ${{ steps.argparse.outputs.sdk_resource }}
+        echo build_list: ${{ steps.argparse.outputs.build_list }}
+        echo target: ${{ steps.argparse.outputs.target }}
+        echo platform: ${{ steps.argparse.outputs.platform }}
+        echo jdk_version: ${{ steps.argparse.outputs.jdk_version }}
+        echo jdk_impl: ${{ steps.argparse.outputs.jdk_impl }}
+
+  runBuild:
+    runs-on: ${{ matrix.platform }}
+    needs: parseComment
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.parseComment.outputs.build_parameters) }}
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Wondered if this checkout is necessary?

Reply:
It is a line that is also in [`PushOrTestPRBuild.yml`](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/.github/workflows/PushOrPRtestbuild.yml#L32) for running tests.

Reply:
Yes, I thought about it some more and realized that this step is probably kind of redundancy for situations which doesn't use default repos.

Reply:
The checkout is necessary, otherwise the job runs into an error.
`Error: Unable to locate executable file: /home/runner/work/openjdk-tests/openjdk-tests/get.sh. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.`

Reply:
I just tried sophia-guo/run-aqa@PR alongside removing the checkout (commit [
ea2f2e6](https://github.com/AdoptOpenJDK/openjdk-tests/pull/2222/commits/ea2f2e65340feda9d4ec7a9a24b225fdefea3a82)). I encounter the same error.
[https://github.com/Icohedron/openjdk-tests/runs/1914866085?check_suite_focus=true](https://github.com/Icohedron/openjdk-tests/runs/1914866085?check_suite_focus=true)

Reply:
Yes, I just updated run-aqa it should be fine. https://github.com/sophia-guo/openjdk-tests/runs/1919789165?check_suite_focus=true

Reply:
With or without the checkout should be fine when the PR https://github.com/AdoptOpenJDK/run-aqa/pull/70 is merged and v1 is updated.

Reply:
I see the PR has been merged. Is v1 updated?

Reply:
Just updated v1.